clippy_lints/src/returns.rs

```diff
@@ -292,7 +295,7 @@ fn last_statement_borrows<'tcx>(cx: &LateContext<'tcx>, expr: &'tcx Expr<'tcx>)
         {
             ControlFlow::Break(())
         } else {
-            ControlFlow::Continue(Descend::from(!expr.span.from_expansion()))
+            ControlFlow::Continue(Descend::from(!e.span.from_expansion()))
```

Comment:
I missed this part and spent far too long wondering how the changes above this fixed the issue 😅

Reply:
Awwww, I made commits separate specifically for this :v

Reply:
Yeah thanks for that, I ended up spotting it because I did a `git log` and saw the separate fix commit